kuma/javascript/src/contributors.jsx

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as React from 'react';
+
+import { gettext } from './l10n.js';
+
+import ContributorsIcon from './icons/contributors.svg';
+
+type Props = {
+    contributors: Array,
+    profileBaseURL: String
+};
+
+export default function Contributors(props: Props) {
+    return (
+        <section className="contributors-sub">
+            <ContributorsIcon />
+            <header>
+                <h4>{gettext('Contributors to this page:')}</h4>
+            </header>
+            <ul>
+                {/*
+                 * TODO: These are temporarily pointing to the wiki domain,
```

Comment:
Not so temporary, is it?
After all, the profiles are and always will in the context of the wiki. 
I think you can simply delete this multi-line comment.